bin/osg-notify

```diff
@@ -203,8 +197,8 @@ def main():
                     emails.add(contact['Email'])
         recipients.update(emails)
 
-    with open(args.message, 'r') as fp:
-        contents = fp.read()
+    with open(args.message, 'rb') as fp:
+        contents = fp.read().decode('utf-8', errors='replace')
```

Comment:
The point of opening binary and then decoding is to handle utf-8 encoding errors, i take it?

Reply:
Exactly.